lib/cli/stats.js

```diff
@@ -159,48 +159,67 @@ async function setMainPath(templateType, templateName) {
   return filePath;
 }
 
+function percentageRoundTwo(num, den) {
+  return den > 0 ? Number(((num / den) * 100).toFixed(2)) : 0;
```

Comment:
[COMMENT] 
Cutoff String value at 2 dp. Agree don't need to be too precious about rounding as for reporting, not exact accounting

Nice safety check with den > 0 to avoid dividing by 0.